packages/eslint-plugin-calcite-components/README.md

```diff
@@ -49,6 +49,10 @@ This rule helps prevent usage of specific events and allows suggesting alternati
 
 This rule catches props/attributes that should be in the encapsulated HTML structure and not on the host element.
 
+- [`@esri/calcite-components/enforce-ref-last-prop`](./docs/enforce-ref-last-prop.md)
+
+This updates `ref` usage to work around a Stencil bug where ref is called in the specified order and not after initializing element with all its attributes/properties. This can cause attributes/properties to be outdated by the time the callback is invoked. This rule ensures the `ref` attribute is ordered last in a JSXElement to keep it up-to-date.
```

Comment:
The `this updates ref usage...` part doesn't really apply here because the rule isn't changing the code. I think it'll read better if we flip it a bit where we describe what the rule does `This rule ensures...` and then follow it with
 
`This updates ref usage to workaround is a workaround for a Stencil bug (<add link to bug*>) where ref is...`